### ISSUE TITLE
Properties are not passed to web component

### ISSUE THREAD

**Issue by kcleong** [EranGrin/vue-web-component-wrapper#6]
When using this wrapper properties are not passed to the web component. 

I've created an example project with a regular web component (using `defineCustomElement`) and one using this wrapper. For the regular component the properties are passed but using the wrapper no properties seem to be passed.

Example project: 

- https://github.com/kcleong/example-vue-webcomponent/
- https://github.com/kcleong/example-vue-webcomponent/blob/main/index.html
- https://github.com/kcleong/example-vue-webcomponent/blob/main/src/main.js
- https://github.com/kcleong/example-vue-webcomponent/blob/main/src/App.ce.vue

```html
<!-- Using defineCustomElement -->
<custom-app-one msg="Hello web component! #1"></custom-app-one>

<!-- Using vue-web-component-wrapper -->
<custom-app-two msg="Hello web component! #2"></custom-app-two>
```

Output:

<img width="1109" alt="Screenshot 2023-08-01 at 09 34 54" src="https://github.com/EranGrin/vue-web-component-wrapper/assets/201650/d512284e-93b2-46f4-bdff-5223aa1f4936">

In the above screenshot I would expect `msg="Hello web component! #2"` to be passed to `App.ce.vue`, however no property is passed.

**Comment by EranGrin** [comment]
Check out this example with 'msg' prop from the web component 
https://stackblitz.com/edit/vue-web-component-wrapper-bja76q?file=demo-app-vite%2Fsrc%2FApp.vue,demo-app-vite%2Findex.html&startScript=vite-demo

**Comment by EranGrin** [comment]
@kcleong I'm closing this issue, as you can see in the example link, it does work with props

**Comment by kcleong** [comment]
@EranGrin this issue has been solved since I've upgraded from version `1.0.5` to `1.1.1`.
